BackendApartmentReservation/BackendApartmentReservation/DataContracts/DataTransferObjects/Requests/RegisterRequest.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BackendApartmentReservation.DataContracts.DataTransferObjects.Requests
+{
+    public class RegisterRequest
+    {
+        [Required(AllowEmptyStrings = false)]
+        public string FirstName { get; set; }
+
+        [Required(AllowEmptyStrings = false)]
+        public string LastName { get; set; }
+
+        [Required(AllowEmptyStrings = false)]
+        [EmailAddress]
```

Comment:
That's what I was talking about, good job! Now, using these attributes allows you to avoid custom regexes and validation that you are doing in the controller